src/ngraph/pass/opset1_upgrade.hpp

```diff
@@ -0,0 +1,39 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/pass/pass.hpp"
+
+namespace ngraph
+{
+    namespace pass
+    {
+        class Opset1Upgrade : public NodePass
+        {
+        public:
+            ///
+            /// \brief    Constructor for the Opset 1 transformation pass.
+            ///
+            /// \details  This transformation pass iterates over all nodes in a graph and updates
+            /// opset version 0 ops
```

Comment:
nit: clang-format probably messed up format here. Maybe break up the lines yourself.

Reply:
Fixed